crates/oxc_minifier/src/peephole/substitute_alternate_syntax.rs

```diff
@@ -1232,59 +1232,6 @@ mod test {
         test_same("new Int8Array(0, a)");
     }
 
-    #[test]
-    #[ignore]
-    fn test_split_comma_expressions() {
-        // Don't try to split in expressions.
-        test_same("while (foo(), !0) boo()");
-        test_same("var a = (foo(), !0);");
-        test_same("a = (foo(), !0);");
-
-        // Don't try to split COMMA under LABELs.
-        test_same("a:a(),b()");
-        test("1, 2, 3, 4", "1; 2; 3; 4");
-        test("x = 1, 2, 3", "x = 1; 2; 3");
-        test_same("x = (1, 2, 3)");
-        test("1, (2, 3), 4", "1; 2; 3; 4");
-        test("(x=2), foo()", "x=2; foo()");
-        test("foo(), boo();", "foo(); boo()");
-        test("(a(), b()), (c(), d());", "a(); b(); c(); d()");
-        test("a(); b(); (c(), d());", "a(); b(); c(); d();");
-        test("foo(), true", "foo();true");
-        test_same("foo();true");
-        test("function x(){foo(), !0}", "function x(){foo(); !0}");
-        test_same("function x(){foo(); !0}");
-    }
-
-    #[test]
-    fn test_comma1() {
-        test_same("x, y");
-    }
-
-    #[test]
-    fn test_comma2() {
-        test("a, a()", "a, a()");
-        test("a, a?.()", "a, a?.()");
-    }
-
-    #[test]
-    fn test_comma3() {
-        test("x, a(), b()", "x, a(), b()");
-        test("x, a?.(), b?.()", "x, a?.(), b?.()");
-    }
-
-    #[test]
-    fn test_comma4() {
-        test("a(), b()", "a(), b()");
-        test("a?.(), b?.()", "a?.(), b?.()");
-    }
-
-    #[test]
-    fn test_comma5() {
-        test("a(), b(), 1", "a(), b(), 1");
-        test("a?.(), b?.(), 1", "a?.(), b?.(), 1");
-    }
-
```

Comment:
I guess we don't need these tests unless we normalize comma expressions into statements before running other compressions.